code/modules/power/tesla/coil.dm

```diff
@@ -14,20 +14,13 @@
 	circuit = /obj/item/circuitboard/machine/tesla_coil
 
 	var/zap_flags = ZAP_MOB_DAMAGE | ZAP_OBJ_DAMAGE
-	var/power_loss = 2
```

Comment:
Why

Reply:
it was being used for both versions to decrease the point gen for the research coils. I thought it was redundant, I can put it back if you'd like